app/helpers/application_helper.rb

```diff
@@ -41,4 +41,8 @@ def flash_class(level)
     when "alert" then "alert alert-warning"
     end
   end
+
+  def og_tag(type, options = {})
```

Comment:
I would prefer that this be given the full name `open_graph_tag` so that future contributors who don't know what it is will have an easy time. :)